src/components/Header/Header.scss

```diff
@@ -0,0 +1,87 @@
+@import '../../variables.scss';
+@import '../../mixins.scss';
+
+.header {
+    width: 100%;
+    height: 300px;
+    position: relative;
+
+    .header_cover {
+        width: 100%;
+        height: 100%;
+        background-image: url(moviecover.jpeg);
+        filter: blur(2px);
+        z-index: -1;
+
+        @include absoluteToTopLeft
+    }
+
+    .header_cover--black {
+        width: 100%;
+        height: 100%;
+        background-color: rgba(0, 0, 0, 0.4);
+
+        @include absoluteToTopLeft
+    }
+
+    .header_logo {
+        // position: absolute;
+        // top: 20px;
+        // left: 60px;
+
+        @include absoluteByArgument(20px, unset, unset, 60px);
+
+        color: $magentaPink;
+
+        span {font-weight: bold;}
+    }
+
+    .header_addMovieButton {
+        // position: absolute;
+        // top: 20px;
+        // right: 60px;
+
+        @include absoluteByArgument(20px, 60px);
+
+        color: $magentaPink;
+        
+    }
+
+    .header_searchWrapper {
+        width: 80%;
+        height: 100%;
+        margin: auto;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+
+        h2 {
+                margin-top: 0;
+                text-transform: uppercase;
+                color: white;
+                font-weight: normal;
+            }
+
+        .header_searchWrapper_inputContainer {
+            width: 100%;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+
+            input {
+                width: calc(80% - 10px);
+                margin-right: 10px;
+            }
+
+            button {
+                width: 20%;
+                height: 35px;
+            }
+        }
+    }
+}
```

Comment:
The symptom of that many curly braces after each other might be a sign to flatten things up a bit :D